Specs/DateTime/Portuguese/DateTimeModel.json

```diff
@@ -1902,5 +1902,79 @@
         }
       }
     ]
+  },
+  {
+    "Input": "Eu voltarei da tarde",
+    "Context": {
+      "ReferenceDateTime": "2019-08-07T00:00:00"
+    },
+    "NotSupported": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "tarde",
+        "Start": 15,
+        "End": 19,
+        "TypeName": "datetimeV2.timerange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "TEV",
+              "type": "timerange",
+              "start": "16:00:00",
+              "end": "20:00:00"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Eu voltarei da noite",
```

Comment:
Similarly, please replace with cases for "à noite", "na noite", and "de noite".

Reply:
added